File: Python/Program/DPR.py
class DPR:
    __id = 0
    __name = ""
    __bidang = ""
    __partai = ""

    def __init__(self, id = 0, name = "", bidang = "", partai = ""):
        self.__id = id
        self.__name = name
        self.__bidang = bidang
        self.__partai = partai

    def get_id(self):
        return self.__id
    
    def set_id(self, id):
        self.__id = id  # Mengubah nilai __id
    
    def get_name(self):
        return self.__name
    
    def set_name(self, name):
        self.__name = name  # Mengubah nilai __name
    
    def get_bidang(self):
        return self.__bidang
    
    def set_bidang(self, bidang):
        self.__bidang = bidang  # Mengubah nilai __bidang
    
    def get_partai(self):
        return self.__partai
    
    def set_partai(self, partai):
        self.__partai = partai  # Mengubah nilai __partai

File: Java/Program/DPR.java
package Program;
public class DPR
{
    private int id;
    private String name;
    private String bidang;
    private String partai;

    public DPR(){
        this.id = 0;
        this.name = "";
        this.bidang = "";
        this.partai = "";
    }

    public DPR(int id, String name, String bidang, String partai){
        this.id = id;
        this.name = name;
        this.bidang = bidang;
        this.partai = partai;
    }

    public int getID(){
        return this.id;
    }

    public void setID(int id){
        this.id = id;
    }

    public String getName(){
        return this.name;
    }

    public void setName(String name){
        this.name = name;
    }

    public String getBidang(){
        return this.bidang;
    }

    public void setBidang(String bidang){
        this.bidang = bidang;
    }
    public String getPartai(){
        return this.partai;
    }

    public void setPartai(String partai){
        this.partai = partai;
    }
}